front/components/RenderMarkdown.tsx

```diff
@@ -64,7 +123,13 @@ function addClosingBackticks(str: string): string {
   return str;
 }
 
-export function RenderMarkdown({ content }: { content: string }) {
+export function RenderMarkdown({
```

Comment:
NIT: we could make `message` optional so that this component can be used in another context. `CiteBlockWrapper` would just return the default if there's no `message`. 

If you want to keep mandatory as is then I suggest `RenderMarkdown` should be renamed as `RenderMessagMarkdown` and the component moved in components/assistant. 

WDYT?